src/imagery/i.modis/i.modis.html

```diff
@@ -339,7 +339,7 @@ <h2>AUTHOR</h2>
 Luca Delucchi, Initial version: Google Summer of Code 2011; subsequently updated by further authors
 
 <h2>SOURCE CODE</h2>
-<p>Available at: <a href="https://github.com/OSGeo/grass-addons/tree/master/grass7/imagery/i.modis">i.modis source code</a> (<a href="https://github.com/OSGeo/grass-addons/commits/master/grass7/imagery/i.modis">history</a>)</p>
+<p>Available at: <a href="https://github.com/OSGeo/grass-addons/tree/master/grass8/imagery/i.modis">i.modis source code</a> (<a href="https://github.com/OSGeo/grass-addons/commits/grass8/src/imagery/i.modis">history</a>)</p>
 
 <!--
 <p>
```

Comment:
If this is a full HTML file since there isn't any real module, and is the homepage for the subfolder addon, why is the header added, with `<body bgcolor="white">` tag opened, but not closed at the end? What is the mechanism for keeping these "meta-manual" pages updated with the builder style?

Reply:
All manual pages belonging to a GRASS module are generated/completed during compilation, hence header/footer are auto-generated.
Only the overview pages (e.g., [i.modis](https://grass.osgeo.org/grass8/manuals/addons/i.modis.html), [i.landsat](https://grass.osgeo.org/grass8/manuals/addons/i.landsat.html), ...) have - so far - to be fully crafted as they are not belonging to a real GRASS module.

A Makefile system tweak would be welcome to apply the same build mechanism as e.g. for [vectorintro.html](https://grass.osgeo.org/grass80/manuals/vectorintro.html) ([code](https://github.com/OSGeo/grass/blob/releasebranch_8_0/vector/vectorintro.html)).